### DISCUSSION TITLE
Logo for the project

### DISCUSSION THREAD

**Discussion by markets** [ondrejbartas/sidekiq-cron#379] [Ideas]
I already discussed this with @ondrejbartas and we agree it would be nice to design some fancy ✨ logo for the project.

Everybody is invited to send suggestions!

**Comment by markets** [comment]
Some proposals:

1) generated via https://logo.com ... I don't have source files, neither the font names :(

![Captura de pantalla 2023-02-10 a las 1 37 27](https://user-images.githubusercontent.com/576701/217971005-92b0120d-8ec7-4b04-8545-9d1acebb38a9.png)

2) created in Figma 🙏🏼 😅 sorry I'm not a designer, just playing 🎨

![Sidekiq Cron logo copia](https://user-images.githubusercontent.com/576701/220493921-fb653344-4f6c-4d9f-880b-a6f9863af6e6.png)

3) another variation

![Captura de pantalla 2023-02-25 a las 2 39 13](https://user-images.githubusercontent.com/576701/221329126-c18551ee-ded3-4333-86dc-259bc23e42eb.png)

**Reply by markets** [reply]
@ondrejbartas @honzasterba @Linell @engwan @petergoldstein @danilogco @serprex did you like any of them? or do you have any better ideas/skills 😅? Proposals are really welcome 🙌🏼

**Reply by danilogco** [reply]
If i can vote I would vote for the first one :D but I'm not really good with this of art hahahah

**Reply by danilogco** [reply]
Maybe if you do use the colors of the sidekiq original logo it would get more attention and faster perception

**Reply by markets** [reply]
Couple of variations with Sidekiq "red":

<img width="795" alt="Captura de pantalla 2023-03-01 a las 1 18 06" src="https://user-images.githubusercontent.com/576701/222013138-f6a33b5d-43ee-48e0-a445-c80dedd265c1.png">
<img width="949" alt="Captura de pantalla 2023-03-01 a las 1 22 03" src="https://user-images.githubusercontent.com/576701/222013340-b7ffbc40-656f-4987-a6b7-7a60c3420faf.png">

**Reply by danilogco** [reply]
The first with blue or the second with red so :dancing_women:

**Comment by petergoldstein** [comment]
I don't love 2 - the other two are fine.

**Comment by Linell** [comment]
I like the second original variation and then the black text on the red background.

**Reply by markets** [reply]
Yeah! Those are also my preferred ones ... Although I'm starting to like more the one with red (Sidekiq's red) background. It's more "impressive" at first look ⚡ 😸 and quite Rubish 💎 

Here the cover and the square versions:

<img width="757" alt="Captura de pantalla 2023-03-02 a las 20 16 46" src="https://user-images.githubusercontent.com/576701/222529328-8afb7315-c55a-4b33-b7a0-1c5140e92dfd.png">


(🙏🏼 sorry if there is any designer reading this! I have no idea what I'm doing)

**Reply by markets** [reply]
Here your go @Linell https://github.com/sidekiq-cron/sidekiq-cron/pull/402

**Reply by markets** [reply]
Final version 🚀 

![cover](https://user-images.githubusercontent.com/576701/222852823-46d3b779-a30b-49a5-b148-cb0d65312e4b.png)

**Comment by honzasterba** [comment]
no prefference, any of them is fine
